packages/frontend-core/src/themes/midnight.css

```diff
@@ -17,5 +17,8 @@
     --modal-background: var(--spectrum-global-color-gray-50);
     --drop-shadow: rgba(0, 0, 0, 0.25) !important;
     --spectrum-global-color-blue-100: rgba(35, 40, 50) !important;
+
+    --spectrum-alias-background-color-secondary: var(--spectrum-global-color-gray-75);
+    --spectrum-alias-background-color-primary: var(--spectrum-global-color-gray-100);
```

Comment:
These are already the default values for these variables - we don't need to redeclare them here.